model/workflow.go

```diff
@@ -19,13 +19,21 @@ import (
 	"fmt"
 )
 
+// ActionMode specifies how actions are to be performed.
+type ActionMode string
+
 const (
-	// DefaultExpressionLang ...
-	DefaultExpressionLang = "jq"
-	// ActionModeSequential ...
+	// ActionModeSequential specifies actions should be performed in sequence
 	ActionModeSequential ActionMode = "sequential"
-	// ActionModeParallel ...
+
+	// ActionModeParallel specifies actions should be performed in parallel
 	ActionModeParallel ActionMode = "parallel"
+)
+
+const (
```

Comment:
is there a reason to break it into two `const` blocks?

Reply:
same as https://github.com/serverlessworkflow/sdk-go/pull/97#discussion_r987872135